### PR TITLE
Options to hide shapes columns in large datasets in FeatureTableFX

### PR DESCRIPTION
**Pull request by robinschmid**
Automatically applies when a new feautre list is loaded.
image

### CODE REVIEW

**Comment by robinschmid** [comment]
image

File: src/main/java/io/github/mzmine/modules/visualization/featurelisttable_modular/FeatureTableFX.java [hunk 1]
```diff
@@ -182,6 +184,49 @@ public FeatureTableFX() {
...
+  /**
+   * @return the maximum samples before deactivating the shapes columns
+   */
+  public int getMaximumSamplesForVisibleShapes() {
+    return parameters.getValue(FeatureTableFXParameters.deactivateShapesGreaterNSamples);
+  }
+
+  private void setShapeColumnsVisible(boolean state) {
+    setVisible(ColumnType.ROW_TYPE, FeatureShapeType.class, state);
+    setVisible(ColumnType.ROW_TYPE, FeatureShapeMobilogramType.class, state);
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/visualization/featurelisttable_modular/FeatureTableFX.java:215]: add FeatureShapeMobilogramHeatmapType, ImageType (FeatureTypes)

**Review by SteffenHeu — DISMISSED** [review]

File: src/main/java/io/github/mzmine/modules/visualization/featurelisttable_modular/FeatureTableFX.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/visualization/featurelisttable_modular/FeatureTableFX.java:215]: thought about this again. 
Currently this is called when small sample sets are loaded. 
Not sure if we want to activate FeatureType columns by default for small datasets... 
Even something like 10 samples might make the table unusable.
Maybe we could only activate those by default with 1 or 2 samples?

Comment (SteffenHeu) [reply src/main/java/io/github/mzmine/modules/visualization/featurelisttable_modular/FeatureTableFX.java:215]: i would just deactivate them for large sample sizes and not change the setting for smaller sample sizes

**Pull request merged**